### PR TITLE
TAJO-1792: tajo-cluster-tests is not available when it is used as an external maven module.

### PR DESCRIPTION
**Pull request by hyunsik**
[Review on Reviewable](https://reviewable.io/reviews/apache/tajo/704)

### CODE REVIEW

File: tajo-jdbc/pom.xml [hunk 1]
```diff
@@ -94,6 +101,73 @@
...
+          <artifactId>jasper-compiler</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.mortbay.jetty</groupId>
+          <artifactId>jsp-2.1-jetty</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>com.sun.jersey.jersey-test-framework</groupId>
+          <artifactId>jersey-test-framework-grizzly2</artifactId>
+        </exclusion>
```
Comment (jinossy) [inline tajo-jdbc/pom.xml]: Can you remove netty dependency?
```
<exclusion>
          <artifactId>netty-all</artifactId>
          <groupId>io.netty</groupId>
</exclusion>
```

File: tajo-cluster-tests/pom.xml [hunk 2]
```diff
@@ -57,12 +57,7 @@
           <excludes>
             <exclude>derby.log</exclude>
             <exclude>benchmark/**</exclude>
-            <exclude>src/test/tpch/**</exclude>
-            <exclude>src/test/resources/dataset/**</exclude>
-            <exclude>src/test/resources/queries/**</exclude>
-            <exclude>src/test/resources/results/**</exclude>
-            <exclude>src/main/resources/META-INF/services/*</exclude>
-            <exclude>src/main/resources/webapps/static/js/*</exclude>
+            <exclude>src/test/resources/tpch/**</exclude>
```
Comment (jinossy) [inline tajo-cluster-tests/pom.xml]: please remove netty dependency in hadoop-hdfs

File: tajo-jdbc/pom.xml (hunk 1 above)
Comment (hyunsik) [inline tajo-jdbc/pom.xml]: fixed

File: tajo-cluster-tests/pom.xml (hunk 2 above)
Comment (hyunsik) [inline tajo-cluster-tests/pom.xml]: fixed

**Comment by hyunsik** [comment]
I fixed the test failure, and reflected the comments.

**Comment by jinossy** [comment]
+1 LGTM!!

**Pull request closed without merging**